gerrit_changes_to_github.py

```diff
@@ -271,6 +279,83 @@ def is_repository_usable(args):
     return got_gerrit and got_target
 
 
+def get_events_from_gerrit(gerrit_password, ghpa_token):
+    """On success a list of event-dicts are returned. On error, None is returned."""
+
+    event_url = "https://review.spdk.io/a/plugins/events-log/events/"
```

Comment:
Would it make sense to move this and the other URLs out of the code and into a cli-argument?